src/Microsoft.Owin.Security.OpenIdConnect/OpenidConnectAuthenticationHandler.cs

```diff
@@ -352,14 +470,6 @@ protected override async Task<AuthenticationTicket> AuthenticateCoreAsync()
                 // Flow possible changes
                 ticket = securityTokenValidatedNotification.AuthenticationTicket;
 
-                Options.ProtocolValidator.ValidateAuthenticationResponse(new OpenIdConnectProtocolValidationContext()
-                {
-                    ClientId = Options.ClientId,
-                    ProtocolMessage = openIdConnectMessage,
-                    ValidatedIdToken = jwt,
-                    Nonce = nonce
-                });
-
                 if (openIdConnectMessage.Code != null)
                 {
                     var authorizationCodeReceivedNotification = new AuthorizationCodeReceivedNotification(Context, Options)
```

Comment:
It does not fire twice because the Code property will be null from the Code redemption. This needs to remain here so the behavior stays the same for hybrid flow.